packages/lib/src/artifacts/Contracts.ts

```diff
@@ -113,4 +114,14 @@ export default class Contracts {
     }
     return createContract(schema);
   }
+
+  private static getFromPathWithUpgradeable(targetPath: string, contractName: string): Contract {
+    const contract = this.getFromPath(targetPath, contractName);
+    const upgradeableName = `${contractName}Upgradeable`;
+    const upgradeablePath = path.join(Contracts.getLocalBuildDir(), `${upgradeableName}.json`);
+    if (fs.existsSync(upgradeablePath)) {
```

Comment:
In a regular deploy the transpiler doesn't run and an upgradeable version doesn't necessarily exist. We shouldn't set the upgradeable property to the non-upgradeable contract because it isn't safe for upgradeability.

Reply:
We now have to clear execution paths in SDK: regular and upgradeable. They very little code in common and I would like to keep them isolated so modifying one will minimize impact on the other.